modules/ossm-migrating-workloads-using-the-istio-injection-label.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/migrating/checklists/ossm-migrating-cluster-wide-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-migrating-workloads-using-the-istio-injection-label_{context}"]
+= Migrating workloads using the istio injection label
+
+Now you can migrate your workloads from the {SMProduct} 2.6 control plane to the {SMproduct} 3.0 control plane.
+
+[NOTE]
+====
+You can migrate workloads and gateways separately, and in any order. For more information, see "Migrating gateways".
+====
+
+.Procedure
+
+. Update the injection labels on the data plane namespace by running the following command:
++
+[source,terminal]
+----
+$ oc label ns bookinfo istio.io/rev=ossm-3-v1-24-3 maistra.io/ignore-namespace="true" istio-injection- --overwrite=true
+----
++
+The previous example adds two labels and removes one.
++
+The `istio.io/rev=ossm-3-v1-24-3` label ensures that 3.0 proxy gets injected to any new or restarted pods in the namespace. In the previous example, the 3.0 revision is named `ossm-3-v1-24-3`.
++
+The `maistra.io/ignore-namespace: "true"` label ensures that 2.6 control plane stops injecting proxies in this namespace to avoid conflicts between 2.6 and 3.0 side car injectors.
++
+The `istio-injection` label must be removed if it exists because it takes precedence over `istio.io/rev` label. Leaving the `istio-injection=enabled` label applied prevents proxy injection.
```

Comment:
Here we need to explain why we are removing `istio-injection=enabled` when we actually want it there. It's only tmp operation so we need to follow the original content here:
`Because we can't create the default IstioRevisionTag yet. It's necessary to temporarily remove the istio-injection=enabled as it would prevent the proxy injection by 3.0 control plane as the istio-injection label takes precedence over istio.io/rev label.`

Reply:
done